src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -272,9 +271,8 @@ def load_command_table(self, _):
                             server_blob_auditing_policies_operations,
                             client_factory=get_sql_server_blob_auditing_policies_operations) as g:
 
-        g.show_command('show', 'get')
-        g.generic_update_command('update',
-                                 custom_func_name='server_audit_policy_update')
+        g.custom_show_command('show', 'server_audit_policy_show')
+        g.custom_command('update', 'server_audit_policy_update', supports_no_wait=True)
```

Comment:
please add wait command 
See wait_command in https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands